src/reuse/modules/nonUi5/userInteraction.ts

```diff
@@ -346,19 +350,12 @@ export class UserInteraction {
       y: +Number(targetSize.height / 2).toFixed(0) + +Number(targetLocation.y).toFixed(0) + 1
     };
 
-    await browser.performActions([
-      {
-        type: "pointer",
-        id: "finger1",
-        parameters: { pointerType: "mouse" },
-        actions: [
-          { type: "pointerMove", duration: 0, x: sourceCenterLocation.x, y: sourceCenterLocation.y },
-          { type: "pointerDown", button: 0 },
-          { type: "pointerMove", duration: 0, x: targetCenterLocation.x, y: targetCenterLocation.y },
-          { type: "pointerUp", button: 0 }
-        ]
-      }
-    ]);
+    await browser.action("pointer")
```

Comment:
is dragAndDrop now working with UI5?

Reply:
I haven't checked it yet. But will just investigate on this one.
As of now for nonUI5 its working fine.

Reply:
I reinvestigated this one, but its not working with the UI5. The behavior is exactly same as described in this comment - https://github.tools.sap/sProcurement/qmate/issues/285#issuecomment-1977349

Reply:
thanks